tasks/fix_secure_path.json

```diff
@@ -8,6 +8,7 @@
         "type": "Optional[String[1]]",
         "default": "/opt/puppetlabs/bin"
       }
-    }
+    },
+    "private": true
```

Comment:
is that a bugfix and should be done in a dedicated PR?

Reply:
👍 moved to another PR